c2cgeoportal/views/layers.py

```diff
@@ -127,21 +126,21 @@ def _get_layers_for_request(self):
                 )
             )  # pragma: no cover
 
-    def _get_layer_for_request(self):
+    def get_layer_for_request(self):
```

Comment:
Why this?

Reply:
My error. I thought that `_` meant private.

Reply:
shouldnt it ?